src/jit/emit.h

```diff
@@ -241,6 +241,59 @@ DECLARE_TYPED_ENUM(insGroupPlaceholderType,unsigned char)
 }
 END_DECLARE_TYPED_ENUM(insGroupPlaceholderType,unsigned char)
 
+// This is a struct that encapsulates the call instruction register return types.
+// It allows for specifying types for multi-register return calls.
+struct         insCallReturnRegisterTypes
```

Comment:
This is a refactoring that we should discuss. I don't think this is the right place for it, as this information should also be available during the Lowering phase, and IMO should be set during morph, when all the call morphing is done, and where I believe we should abstract the ABI information.